src/twisted/trial/test/pyunitcases.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+
+
+class PyUnitTest(unittest.TestCase):
+    def test_pass(self):
+        pass
```

Comment:
Do we need this method?  Same question for `test_skip`.

I see this code `self.original = pyunitcases.PyUnitTest("test_pass")` but I am not sure how it works.

I am asking, since it looks like they don't have coverage... while the other methods have.

Maybe add a comment/docstring to explain why they are needed, and add a pragma nocover comment to make coverage.py happy :)